KDCoin/SPVApp.py

```diff
@@ -1,49 +1,76 @@
 import requests
-from KDCoin.spvClient import SPVClient
 import json
+from flask import Flask, request
 
-#Broadcast to this transaction to all miners
-#Takes transaction object created by spvClient
-#POST REQUEST
-def postTransaction(transaction_tobeposted):
-    # Validation process here: Check if transaction came from correct sender
-    if (transaction_tobeposted.validate()):
-        print('got into here')
-        json_tobesent = transaction_tobeposted.to_json(transaction_tobeposted.data)
-        print(json_tobesent)
 
-        #Send json to server
-        t1 = requests.post('http://127.0.0.1:8080/createTransaction',
-                       headers = {'Content-Type': 'application/json'},
-                       json = json_tobesent)
+app = Flask(__name__)
 
-#Check acc balance of Specified spvClient
-def checkBalance(accountpublickey):
-    #Check against blockchain of any miner, using accountpublickey
-    publickeyinfo = {'publickey_tobechecked': accountpublickey}
-    json_tobesent = json.dumps(publickeyinfo)
+miners_list = []
 
```

Comment:
please also include client object here:
```client = spvClient.SPVClient()```

This way, you can just use ```client.createTransaction()``` from endpoint data in your ```createTransaction``` endpoint